### DISCUSSION TITLE
Collection actualization whatsapp pics

### DISCUSSION THREAD

**Discussion by Masterblasterstorm** [deckerst/aves#310] [Q&A] [answered]
Hi, is there a possibility to get "start a manually quick collection actualization" (looking for new whatsapp pics or anything else on the device)? It actually scans for new whatsapp pics only sometimes, or when close the app and restart it or when I hold the app icon - > tip on Videos - > and switch to collection, then new whatsapp pics appear immedialy ( or any Pic not from camera) otherwise later. 

Thank you

**Comment by deckerst** [comment, marked as answer]
Aves automatically refreshes and shows new images as soon as the native Media Store on your device discovers them. That's why there is no manual actualization mechanism in Aves. If you don't see new pictures right away, it's either:
1) the Media Store on your device did not discover yet these images and is not updated, so Aves cannot know about them either,
2) there's a bug in Aves that prevents it to react to the Media Store update.

If it's 1), there's nothing Aves can do. I believe that kind of issue happens especially on older devices. Sometimes it just takes a little time.
If it's 2), please file a bug report. After you witness the issue you described, please follow the instructions in `About` > `Bug Report`.

**Reply by Masterblasterstorm** [reply]
Hi Deckerst, I have tried it on an other device and I can confirm variant 1. It's not the app.

Thank you for your support.
